identity/handler.go

```diff
@@ -74,6 +77,10 @@ func (h *Handler) RegisterAdminRoutes(admin *x.RouterAdmin) {
 	admin.POST(RouteValidate, h.validate)
 	// fandom-end
 	admin.PUT(RouteItem, h.update)
+
+	//fandom-start - allow admins to remove multifactor authentication for an identity
```

Comment:
Just in case someone will be looking for `// fandom`, as we do everywhere else
```suggestion
	// fandom-start - allow admins to remove multifactor authentication for an identity
```